### PR TITLE
Configuration File for XP-Pen Artist 16 (2nd Gen)

### PR DESCRIPTION
**Pull request by SerialBoxes**
config file for the xp-pen artist 16 2nd gen based off of the config files for the other tablets in this same product family and tested on my device.

### CODE REVIEW

**Comment by jamesbt365** [comment]
Please add an entry to the tablets.md file.

File: OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 16 (2nd Gen).json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "Name": "XP-Pen Artist 16 (2nd Gen)",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 340.99,
+      "Height": 191.81,
+      "MaxX": 68198.0,
+      "MaxY": 38362.0
+    },
+    "Pen": {
+      "MaxPressure": 8192,
```
Comment (jamesbt365) [inline OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 16 (2nd Gen).json:11]: Are you sure this isn't meant to be set to 8191?

Comment (SerialBoxes) [reply OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 16 (2nd Gen).json:11]: you're right I forgot about 0 indexing I'll fix it, gimme a sec

**Comment by SerialBoxes** [comment]
I calculated the numbers for the config file using the numbers on this product page: https://www.xp-pen.com/product/1203.html

**Review by jamesbt365 — APPROVED** [review]

**Comment by jamesbt365** [comment]
https://cdn.discordapp.com/attachments/615611007951306863/1061285515065827328/verification_log.json

**Pull request merged**